### PR TITLE
fix: indexer build

### PR DESCRIPTION
**Pull request by ponderingdemocritus**
Indexer was failing because of apibara deps and container deps.

Simple fix so it builds and runs

### CODE REVIEW

File: cairo [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 439da05a031c2eda263c4ce12d0b71d20f38205f
+Subproject commit 679d0472d03177df24ad46058cf389725c2914be
```
Comment (tarrencev) [inline cairo:1]: we need to keep cairo on the alpha 7 commit hash

File: crates/dojo-indexer/Cargo.toml [hunk 2]
```diff
@@ -9,9 +9,9 @@ edition = "2021"
 actix-web = "4.3.0"
 actix-cors = "0.6.4"
 anyhow.workspace = true
-apibara-core = { git = "https://github.com/apibara/apibara", rev = "5afaa6db00f2d1b36825215bbf894f0341121d39" }
-apibara-sdk = { git = "https://github.com/apibara/apibara", rev = "5afaa6db00f2d1b36825215bbf894f0341121d39" }
-apibara-starknet = { git = "https://github.com/apibara/apibara", rev = "5afaa6db00f2d1b36825215bbf894f0341121d39" }
```
Comment (tarrencev) [inline crates/dojo-indexer/Cargo.toml:14]: we can use a cargo patch here to temporarily use the fork

cc @fracek

**Comment by tarrencev** [comment]
the devcontainer ci is failing due to the previous protostar change

File: cairo (hunk 1 above)
Comment (ponderingdemocritus) [reply cairo:1]: yep, old branch - this is the commit, isn't it? 

81c4eb942dace6849a8067dc43263da99579bef8

**Pull request closed without merging**